### PR TITLE
fix: Kill inx and hornet after cluster tests on macOS

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: tools/cluster/tests/cluster.go [hunk 1]
```diff
@@ -64,7 +64,14 @@ func newCluster(t *testing.T, opt ...waspClusterOpts) *cluster.Cluster {
...
+		stopCluster(clu)
+	})
 
 	return clu
 }
+
+func stopCluster(clu *cluster.Cluster) {
+	clu.Stop()
+	l1.Stop()
+}
```
Comment (jorgemmsilva) [inline tools/cluster/tests/cluster.go:77]: this is a bad idea, the L1 is kept alive though a "cluster test" run. (many test cases are executed on the same network)

File: .github/workflows/heavy-tests.yml [hunk 2]
```diff
@@ -247,6 +249,7 @@ jobs:
     name: Test
     runs-on:
       - self-hosted
+      - macos-latest
```
Comment (jorgemmsilva) [inline .github/workflows/heavy-tests.yml:252]: what's the point of adding this? this yml file is only for CI

File: tools/cluster/tests/cluster.go (hunk 1 above)
Comment (howjmay) [reply tools/cluster/tests/cluster.go:77]: umm or I should add the stop function in the end of all tests?

File: .github/workflows/heavy-tests.yml (hunk 2 above)
Comment (howjmay) [reply .github/workflows/heavy-tests.yml:252]: This is for the test. I want to enable cluster test in this PR to check whether anything wrong here. I will remove this once the PR is ready

File: tools/cluster/tests/cluster.go (hunk 1 above)
Comment (jorgemmsilva) [reply tools/cluster/tests/cluster.go:77]: lol no. please re-read my comment.

**Pull request closed without merging**